gui/src/components/Backups.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+    <div>
+        <h4 class="py-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3 py-3">
+            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create Backup
+            </button>
+            <button
+                    type="button"
+                    class="btn btn-primary mx-1"
+                    @click="viewRestoreHistory()"
+            >View Restores
+            </button>
+        </div>
+        <table class="table text-center table-bordered">
+            <thead class="table-secondary">
```

Comment:
At first glance, it's difficult to distinguish that there are 2 tables displayed in the page. I am wondering if the cluster info table header color should be set to a different color from the backup table.

Reply:
The screen I am talking about.
![Screen Shot 2020-07-01 at 18 07 14](https://user-images.githubusercontent.com/3835021/86230098-958ca880-bbcb-11ea-838b-7323908ac3e0.png)

Reply:
Good idea. Differentiating by color or structure. 
Having no lines and colors like this might be good. 
![tokens](https://user-images.githubusercontent.com/1052473/86303414-b514e700-bc46-11ea-850a-ed3d99736efa.png)

Or having different structure like this ? (not sure how to remove the header in Markdown)

|    |    |
|-----|--------|
|Cluser ID| test_cluster-xxx    |
|Target IPs  | ["127.0.0.1"]    |
|Keyspaces | ["x", "y", "z"]  |

Reply:
Nice, I like the screenshot layout. 
By not putting the cluster info (cluster id, target ips, keyspaces,etc) inside a table, it will put less emphasis on it. The result is that the backup table should pop out.

Reply:
In terms of keeping the table horizontal, here are a few possible layouts. Do you like any of the following or should I try something else?

v1: Small table
![Screen Shot 2020-07-02 at 15 38 51](https://user-images.githubusercontent.com/46988771/86325037-64b77c80-bc7a-11ea-8558-1b07977a29be.png)
v2: Borderless table
![Screen Shot 2020-07-02 at 15 39 13](https://user-images.githubusercontent.com/46988771/86325058-6b45f400-bc7a-11ea-83f8-dc26bddc7bd3.png)
v3: Normal table, no color
![Screen Shot 2020-07-02 at 15 39 54](https://user-images.githubusercontent.com/46988771/86325066-6da84e00-bc7a-11ea-84c9-750359e2ae11.png)

Reply:
I vote for v2.